crescent/ext/tasks/cron.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime
+from typing import Callable, Sequence
+
+from croniter import croniter
+
+from crescent.ext.tasks.task import Task, TaskCallbackT, link_task
+from crescent.internal import MetaStruct
+
+__all__: Sequence[str] = ("cronjob", "Cronjob")
+
+
+class Cronjob(Task):
+    def __init__(self, cron: str, callback: TaskCallbackT) -> None:
+        self.cron = croniter(cron, datetime.now())
+
+        super().__init__(callback)
+
+    def next_iteration(self) -> float:
+        call_next_at: datetime = self.cron.get_next(datetime)
+        time_to_next = call_next_at - datetime.now()
+        return time_to_next.total_seconds()
```

Comment:
Docstring?

Reply:
Inherits from an ABC so base class has the docstring